labels.yml

```diff
@@ -16,20 +16,40 @@ labels:
     color: 'f5f5f5'
 
   - name: monitoring
-    description: "This issue should be solved, but it should be monitored for a period before being closed"
+    description: "This issue should be solved, but let's keep an eye on it for a while before we close it."
     color: '357399'
 
   - name: performance
-    description:
+    description: "This issue significantly affects current performance, or will when we scale."
     color: '208116'
 
+  - name: operations
+    description: Anything related to our internal IT systems.
+    color: '262626'
+
+  - name: infrastructure
+    description: Product-related tooling.
+    color: '56a39f'
+
+  - name: debt
+    description: "When we originally built this feature, we knew we'd need to improve it later."
+    color: 'b51e1e'
+
   - name: refactor
-    description:
+    description: "New customer requirements, software updates, or changing business needs are motivating us to improve this."
     color: '627ada'
 
   - name: security
-    description:
-    color: '3b4047'
+    description: A significant customer-facing security issue.
+    color: '191a47'
+
+  - name: dev_research
+    description: "We know the outcome. How might we solve it?"
+    color: '9083bd'
+
+  - name: client_services
```

Comment:
> How would you distinguish enterprise from ops?

Most of the `ops` work I'm doing right now is actually `client_services` + `operations`, since its only relevant to CMS. 

What I'm looking for a label for is something that involves all future enterprise customers, like improving our docker base image, which could be `enterprise` + `operations`, or improving our enterprise onboarding flow.

> enterprise could be interpreted to include enterprise features that SaaS customers might also use

Point taken.

Other possibilities include `on_premise` and `single_tenant`. Single tenant would actually be fairly useful, because we literally have that as a flag in our codebase, and a decent amount of behavior depends on that flag. Of course, that flag is set when RAILS_ENV=enterprise, which is always true when its running on premise lol.

I think a pretty concrete way to limit the scope of this label would be to describe it as only relevant when `RAILS_ENV=enterprise`. At the same time, I see your point, and would be fine with just sticking with `client_services` for now. that label will probably cover most open CMS issues we have, and since we wont have enterprise customers for a while we probably won't even need any other labels.